docs/react-v9/contributing/rfcs/rfc-template.md

```diff
@@ -5,17 +5,20 @@ An RFC can be anything. A question, a suggestion, a plan. The purpose of this te
 
 Tips for writing a successful RFC:
 
-- Simple plain words that make your point, fancy words obfuscate
+- Favor simple language that make your point, fancy words obfuscate // Speak plainly.
 - Try to stay concise, but don't gloss over important details
+- Feel free to add necessary diagrams in the RFC assets folder
 - Try to write a neutral problem statement, not one that motivates your desired solution
 - Remember, "Writing is thinking". It's natural to realize new ideas while writing your proposal
 -->
 
 ---
 
 _List contributors to the proposal here_
+_List effected stake holders_
```

Comment:
I think a bit more clarity here might be good. I'm assuming we don't want to list internal stake holders but more so types of users or contributors of the library? An example might be an RFC impacting people who are making custom themes. Is that the idea here or do we mean actual business level stake holders?

Reply:
Yeah, general personas are good.

But I do think listing individual people would help. I feel like one of the pain points for RFCs is getting audience participation. Listing them out might help drive engagement.

Reply:
I think it's ok if we list GitHub usernames, but not internal team names for security reasons, which is what I think @brandonthomas was talking about here.